controllers/nstemplateset/nstemplateset_controller_test.go

```diff
@@ -1102,133 +894,130 @@ func TestReconcileUpdate(t *testing.T) {
 				}).
 				HasConditions(Updating())
 			AssertThatCluster(t, fakeClient).
-				HasResource("for-"+spacename, &quotav1.ClusterResourceQuota{},
-					WithLabel(toolchainv1alpha1.TemplateRefLabelKey, "advanced-clusterresources-abcde12"),
-					WithLabel(toolchainv1alpha1.TierLabelKey, "advanced")). // upgraded
-				HasResource(spacename+"-tekton-view", &rbacv1.ClusterRoleBinding{},
-					WithLabel(toolchainv1alpha1.TemplateRefLabelKey, "advanced-clusterresources-abcde11"),
-					WithLabel(toolchainv1alpha1.TierLabelKey, "advanced"))
-
-			for _, nsType := range []string{"stage", "dev"} {
-				AssertThatNamespace(t, spacename+"-"+nsType, r.Client).
-					HasNoOwnerReference().
-					HasLabel(toolchainv1alpha1.SpaceLabelKey, spacename).
-					HasLabel(toolchainv1alpha1.TemplateRefLabelKey, "advanced-"+nsType+"-abcde11"). // not upgraded yet
-					HasLabel(toolchainv1alpha1.TierLabelKey, "advanced").
-					HasLabel(toolchainv1alpha1.TypeLabelKey, nsType).
-					HasLabel(toolchainv1alpha1.ProviderLabelKey, "codeready-toolchain").
-					HasResource("crtadmin-pods", &rbacv1.RoleBinding{}).
-					HasResource("exec-pods", &rbacv1.Role{}).
-					HasResource("crtadmin-view", &rbacv1.RoleBinding{})
-			}
+				HasResource("for-"+spacename, &quotav1.ClusterResourceQuota{}).
+				HasNoResource(spacename+"-tekton-view", &rbacv1.ClusterRoleBinding{})
 
-			t.Run("delete ClusterRoleBinding", func(t *testing.T) {
```

Comment:
you should update the name of the sub-test as well
`"update ClusterResourceQuota"` -> `"update CRQ, delete CRB, and delete redundant namespace"`